packages/ckeditor5-dev-env/lib/release-tools/tasks/bump-year.js

```diff
@@ -0,0 +1,95 @@
+/**
+ * @license Copyright (c) 2003-2021, CKSource - Frederico Knabben. All rights reserved.
+ * For licensing, see LICENSE.md.
+ */
+
+'use strict';
+
+const chalk = require( 'chalk' );
+const fs = require( 'fs' );
+const glob = require( 'glob' );
+
+/**
+ * Updates year in all licenses in the provided directory, based on provided glob patterns.
+ *
+ * Replaces:
+ * 	Copyright (c) [initial year]-xxxx
+ *
+ * With:
+ *	Copyright (c) [initial year]-[current year]
+ *
+ * @param {Object} params
+ * @param {String} params.cwd Current working directory from which all paths will be resolved.
+ * @param {Number} [params.initialYear='2003'] Year from which the licenses should begin.
+ * @param {Array} params.globPatterns array of objects, where each object has string property 'pattern',
+ * and optionally 'options' property.
```

Comment:
I know the `params` argument is an object so order of properties does not matter, but to conform to the convention it is better to define optional arguments last. Also there is a typo in the `array` (should be in uppercase), and the type of the `initialYear` should be a string, according to its default value.

```suggestion
 * @param {Array} params.globPatterns An array of objects, where each object has string property `pattern`,
 * and optionally `options` property for this `glob` pattern.
 * @param {String} [params.initialYear='2003'] Year from which the licenses should begin.
```